ui/app/models/pki/issuer.js

```diff
@@ -13,39 +15,57 @@ const validations = {
 };
 
 @withModelValidations(validations)
-export default class PkiIssuerModel extends Model {
-  @attr('string', { readOnly: true }) backend;
+@withFormFields(null, [
+  {
+    default: [
+      'certificate',
+      'caChain',
+      'commonName',
+      'issuerName',
+      'notValidBefore',
+      'serialNumber',
+      'keyId',
+      'allowedUriSans',
+      'notValidAfter',
+    ],
+  },
+  { 'Issuer URLs': ['issuingCertificates', 'crlDistributionPoints', 'ocspServers'] },
+])
+export default class PkiIssuerModel extends PkiCertificateBaseModel {
+  getHelpUrl(backend) {
+    return `/v1/${backend}/issuer/example?help=1`;
+  }
+  @attr('string', { displayType: 'masked' }) certificate;
+  @attr('string', { displayType: 'masked', label: 'CA Chain' }) caChain;
+  @attr('date', {
+    label: 'Issue date',
+  })
+  notValidBefore;
+
   @attr('string', {
-    label: 'Issuer name',
-    fieldValue: 'id',
+    label: 'Default key ID',
+  })
+  keyId;
+
+  @attr({
+    label: 'Subject Alternative Names',
   })
-  name;
+  allowedUriSans;
```

Comment:
looking [here](https://developer.hashicorp.com/vault/api-docs/secret/pki#parameters-1) I see this referred to as `uri_sans`